pages/reference/livepeer-js/Player.en-US.mdx

```diff
@@ -579,6 +579,8 @@ function PlayerComponent() {
 
 Enables automatic upload and playback from decentralized storage providers. Currently supports IPFS CIDs and IPFS/Arweave URLs. Defaults to `true`.
 
+Note: For IPFS HTTP gateway URLs, the player currently only supports “path style” URLs and does not support “subdomain style” URLs. The player will support both styles of URLs in a future update.
```

Comment:
same here

Reply:
Fixed in [ae8328d](https://github.com/livepeer/docs/pull/346/commits/ae8328d1b6f74b171e78692f42ad8a616b07bb81)